### DISCUSSION TITLE
I declared textStyles in 'panda.config.ts' but I want to get a type, is there a way?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/panda#2399] [Q&A]
I want to receive and apply textStyle as a prop of the component, but I want to receive this only as a textStyle that I defined, how do I know the type?

**Comment by ghost** [comment]
`PropertyValue<'textStyle'>` Can I just import it with this?

**Comment by manspaniel** [comment]
Can use `PropertyTypes['textStyle']` if you just want the pure union of text style names.

Not sure if there's a more idiomatic way.

**Reply by ghost** [reply]
I only want to have the added properties, but I'm satisfied with this much. Thank you

**Reply by astahmer** [reply]
we merged [a PR last week](https://github.com/chakra-ui/panda/pull/2347) (not yet published on npm) that changes some internal typings, including this one, you should use `UtilityValues['textStyles']` when that version comes out

it will be a direct union of allowed textStyles, as you need
![image](https://github.com/chakra-ui/panda/assets/47224540/77f8923e-66e6-412a-a06a-89c058796be0)

if you need the same type as what's allowed in `css`, you can use the `SystemStyleObject` type

**Reply by segunadebayo** [reply]
Consider using `SystemStyleObject["textStyle"]` as well.

`SystemStyleObject` is the generic type that refers to the Panda-related style objects.
